### DISCUSSION TITLE
Is it possible to use both Coral and OpenVino?

### DISCUSSION THREAD

**Discussion by miczlo** [blakeblackshear/frigate#5063] [Ask A Question] [answered]
Is it possible to mix the detectors?
I've been using Coral and am running out of fps so could do with additionally utilising the CPU, but I'm not too sure - is it possible?
Looking at the config there's just one model that is defined which kinda makes sense but still I though I'd ask.

**Comment by NickM-27** [comment, marked as answer]
Models can be defined for each detector but only the model path can be changed. Different detectors is not currently supported.

**Reply by miczlo** [reply]
Thanks, that's what I thought.
